src/Components/TodoContainer.js

```diff
@@ -72,13 +86,16 @@ function TodoContainer() {
 
     return (
         <>
-            <button onClick={toggleSortingOrder}
+            <button
+                onClick={toggleSortingOrder}
                 style={{
-                    margin: "16px",
+                    margin: "16px auto",
                     cursor: "pointer",
+                    alignItems: "center",
+                    textIndent: "0",
+                    backgroundColor: "#01BAD7",
                 }}
```

Comment:
we should never use any `style` tags, preferring instead to put all styling inside of `[ComponentName].module.css`, assigned to classes